packages/core/src/main/conversation/ConversationService/ConversationService.ts

```diff
@@ -1169,7 +1169,7 @@ export class ConversationService {
       qualified_users: undefined,
     });
     const {group_id: groupId, qualified_id: qualifiedId} = newConversation;
-    const groupIdDecodedFromBase64 = Decoder.fromBase64(groupId!).asBytes;
+    const groupIdDecodedFromBase64 = Decoder.fromBase64(`${groupId}`).asBytes;
```

Comment:
why this? converting it to a string from a Uint8Array? seems like it should happen a different way.